src/common/main/kotlin/tech/thatgravyboat/skyblockapi/api/profile/items/sacks/SacksData.kt

```diff
@@ -1,13 +1,22 @@
 package tech.thatgravyboat.skyblockapi.api.profile.items.sacks
 
+import kotlinx.datetime.Instant
 import me.owdding.ktcodecs.GenerateCodec
 import tech.thatgravyboat.skyblockapi.generated.SkyblockAPICodecs
+import tech.thatgravyboat.skyblockapi.utils.time.currentInstant
 
-@GenerateCodec
+@GenerateCodec(createCodecMethod = true)
 data class SacksData(
-    var items: MutableMap<String, Int> = mutableMapOf(),
+    var items: MutableList<SackEntry> = mutableListOf(),
 ) {
     companion object {
         internal val CODEC = SkyblockAPICodecs.getCodec<SacksData>()
     }
 }
+
+@GenerateCodec
+data class SackEntry(
+    val id: String,
+    val amount: Int,
+    val lastUpdated: Instant = currentInstant(),
+)
```

Comment:
we should probably make these internal

Reply:
well i can make the lastUpdated internal but i dont see the point